addons/pos_sale/static/src/js/OrderManagementScreen/SaleOrderManagementScreen.js

```diff
@@ -72,6 +72,19 @@ odoo.define('pos_sale.SaleOrderManagementScreen', function (require) {
             SaleOrderFetcher.setPage(1);
             SaleOrderFetcher.fetch();
         }
+        async _loadPartner(saleOrders) {
```

Comment:
Use array of partner id as parameter instead of the order. It will something like:
```js
async _loadPartners(partnerIds) {
```

And to use it:
```js
const missingPartnerIds = saleOrders.map(order => order.partner_id[0]);
await this._loadPartners(missingPartnerIds);
```